src/modules/timeDimensions.js

```diff
@@ -123,3 +124,11 @@ export const getDisabledTimeDimensions = (inputType, program, stage) => {
         }
     }
 }
+
+export const isAoWithTimeDimension = ({ columns }) =>
+    columns.some(
+        ({ dimension, items }) =>
+            DIMENSION_IDS_TIME.has(dimension) &&
+            Array.isArray(items) &&
+            items.length > 0
+    )
```

Comment:
Please triple check this makes sense

Reply:
We have tools for this in the analytics lib. I'll send you a link.

Reply:
@jan I rewritten this and did use helpers from analytics, but not exactly the one you pointed me to....
```js
export const isAoWithTimeDimension = (ao) =>
    layoutGetAllDimensions(ao).some(
        ({ dimensionType, dimension, items }) =>
            (dimensionType === DIMENSION_TYPE_PERIOD ||
                DIMENSION_IDS_TIME.has(dimension)) &&
            Array.isArray(items) &&
            items.length > 0
    )
```